### PR TITLE
docs(addon): better manage alpha/beta providers

### PR DESCRIPTION
**Pull request by davlgd**
Prepare Alpha public access for Keycloak.
Better manage Alpha/Beta message after an add-on creation. One is added for Pulsar.

### CODE REVIEW

**Review by hsablonniere — CHANGES_REQUESTED** [review]

File: src/commands/addon.js [hunk 1]
```diff
@@ -101,6 +101,47 @@ async function create (params) {
...
+  };
+
+  const pulsar = {
+    status: 'beta',
+    postCreateInstructions: [
+      'Learn more about Pulsar on Clever Cloud: https://developers.clever-cloud.com/doc/addons/pulsar/',
+    ].join('\n'),
+  };
+
+  const wipProviders = { keycloak: keycloak, kv: kv, 'addon-pulsar': pulsar };
```
[the remark this answers is not in the document]
Comment (davlgd) [reply src/commands/addon.js:129]: As discussed in sync, I'll directly declare the whole object.

[the remark this answers is not in the document]
Comment (davlgd) [reply src/commands/addon.js:129]: As discussed, I need some variables (`addon.id` for example) to build this object, so I can't just move it. I prefer to let it there if it's ok for you.

**Review by aurrelhebert — APPROVED** [review]

File: src/commands/addon.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (hsablonniere) [reply src/commands/addon.js]: OK, let's go like this for now :+1:

**Review by hsablonniere — APPROVED** [review]
LGTM :+1:

**Pull request merged**